specification/saas/resource-manager/Microsoft.SaaS/preview/2018-03-01-beta/saas.json

```diff
@@ -356,6 +356,445 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SaaS/resources/{resourceName}": {
+      "get": {
+        "tags": [
+          "SubscriptionLevelSaaSResources"
+        ],
+        "operationId": "SaasSubscriptionLevel_Get",
+        "description": "Gets information about the specified Subscription Level SaaS.",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The request returns the resulting SaaS resource.",
+            "schema": {
+              "$ref": "#/definitions/SaasResource"
+            }
+          },
+          "default": {
+            "description": "A response indicating an error.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get saas application": {
+            "$ref": "./examples/saasSubscriptionLevel/SaasGet.json"
+          }
+        }
+      },
+      "put": {
```

Comment:
>put [](start = 7, length = 3)

You are missing the x-ms-long-running annotation.

Reply:
will add